charts/civil-ccd/values.preview.template.yaml

```diff
@@ -1,5 +1,6 @@
 civil-service:
   java:
+    image: 'hmctspublic.azurecr.io/civil/service:pr-3092'
```

Comment:
again, before merging please revert it back

Reply:
Reverted